distributor-api-v1/environments.md

```diff
@@ -16,8 +16,8 @@ You will also have access into the system so it is possible for you to check whe
 
 ## Production environment
 
-* **API base url** - `https://www.mews.li`
-* **Mews applications base url** - `https://www.mews.li`
+* **API base url** - `https://api.mews.com`
+* **Mews applications base url** - `https://app.mews.com`
```

Comment:
@koprivajakub Just for clarification: If this `https://app.mews.com` is `Mews applications base url` shouldn't the Distributor standalone link lead to `https://app.mews.com/distributor/aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee?currency=EUR&mewsRoute=rates&mewsRoom=aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee` instead?

Basically, if it's an app: use `https://app.mews.com`
If you want to download JS or use Distributor API, use https://api.mews.com

Or what's the logic do we want to have?

Reply:
nope, it would not work since this would be at the end CrossOrigin request.

the same way apply for e.g. Navigator where it is calling API on app.mews.com not on api.mews.com

Reply:
This seems to be the only place where we refer to this (at least in this PR), why do we need it here at all?

Reply:
@siroky There is a reference in the UseCases. Where we are describing how to get work with PaymentRequest and CardAuthorization via GuestPortal application. And few more references on where to get the ConfigurationIds etc...